resources/js/Pages/Asisten.vue

```diff
@@ -576,6 +610,63 @@ export default {
         this.menuAllLaporan = $bool;  
     },
 
+    editDescription: function($bool){
+      const globe = this;
+      if ($bool==true) {
+        $(".descEdit").removeClass("hidden");
+        $(".descEdit").addClass("visible");
+        $(".editDescBtn").removeClass("visible");
+        $(".editDescBtn").addClass("hidden");
+        $(".editDescClose").removeClass("hidden");
+        $(".editDescClose").addClass("visible");
+        $(".updateDescBtn").removeClass("hidden");
+        $(".updateDescBtn").addClass("visible");
+        this.formDesc.deskripsi = this.currentUser.deskripsi;
+      }else{
+        $(".descEdit").removeClass("visible");
+        $(".descEdit").addClass("hidden");
+        $(".editDescBtn").removeClass("hidden");
+        $(".editDescBtn").addClass("visible");
+        $(".editDescClose").removeClass("visible");
+        $(".editDescClose").addClass("hidden");
+        $(".updateDescBtn").removeClass("visible");
+        $(".updateDescBtn").addClass("hidden");
+      }
+    },
+
+    updateDeskripsi: function(){
+      const globe = this;
+      this.$axios.post('/updateDesc', this.formDesc).then(response => {
+        
+        console.log(response.data)
```

Comment:
remove all debugging statement like this one console.log() when you want to commit stuff, so remove this one too

Reply:
thank you for reminder bang, i'll remove this debugging statement